ast/rego.peg

```diff
@@ -1,311 +0,0 @@
-{
```

Comment:
💭 Do we have a valid grammar somewhere? Could be useful to library authors processing Rego code.

Reply:
There is still one in the language reference documentation https://github.com/open-policy-agent/opa/blob/master/docs/content/policy-reference.md#grammar